client/blocks/simple-site-rename-form/index.jsx

```diff
@@ -14,12 +14,23 @@ import { connect } from 'react-redux';
 import Card from 'components/card';
 import FormButton from 'components/forms/form-button';
 import FormTextInputWithAffixes from 'components/forms/form-text-input-with-affixes';
+import FormInputValidation from 'components/forms/form-input-validation';
 import ConfirmationDialog from './dialog';
 import FormSectionHeading from 'components/forms/form-section-heading';
 import { requestSiteRename } from 'state/site-rename/actions';
 import { isRequestingSiteRename } from 'state/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
 
+export const getDomainValidationMessage = domain => {
+	if ( ! domain.match( /^[a-z0-9]+$/i ) ) {
+		return translate( 'Domain can only contain letters and numbers' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **85** times:
`translate( 'Usernames can only contain lowercase letters (a-z) and numbers.' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/sites-rename--validation)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).